src/mergePaths.js

```diff
@@ -0,0 +1,31 @@
+import { curry, mergeAll } from 'ramda';
+
+import paths from './paths';
+
+/**
+ * Merge objects under corresponting paths.
+ *
+ * @func mergePaths
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.17.0|v1.17.0}
+ * @category Object
+ * @sig [[k]] -> {k: {a}} -> {a}
+ * @see {@link RA.mergeProps|mergeProps}
+ * @param {!Array} ps The property paths to merge
+ * @param {!Object} obj The object to query
+ * @return {!Object} The object composed of merged properties of obj
+ * @example
+ *
+ * const obj = {
+ *   foo: { fooInner: { fooInner2: 1 } },
+ *   bar: { barInner: 2 }
+ * };
+ *
+ * R.mergePaths(
```

Comment:
Can we go for more compact onel-iner here ?

```javascript
RA.mergePaths([['foo', 'fooInner'], ['bar']], obj); //=> { fooInner2: 1, barInner: 2 }
```

Also not `R` vs `RA` mismatch. Should be `RA`.